apps/cow-tools/src/app/milkman/[network]/order/new/page.tsx

```diff
@@ -98,57 +94,9 @@ function TransactionCard({
 }) {
   const { transactionStatus, setTransactionStatus } = useOrder();
   const network = networkFor(chainId);
-  const router = useRouter();
-
-  const form = useForm();
-  const { sendTransactions } = useRawTxData();
-
-  const { register, setValue, watch, clearErrors } = form;
-
-  useEffect(() => {
-    register("receiverAddress");
-    register("tokenBuy");
-    register("tokenSell");
-  }, []);
-
-  const formData = watch();
 
-  useEffect(() => {
-    if (formData.isValidFromNeeded === false) {
-      setValue("validFrom", null);
-      clearErrors("validFrom");
-    }
-  }, [formData]);
-
-  async function handleOnSubmit(data: FieldValues) {
-    const sellAmountBigInt = BigInt(
-      Number(data.tokenSellAmount) * 10 ** data.tokenSell.decimals,
-    );
-    const priceCheckersArgs = priceCheckerInfoMapping[
-      data.priceChecker as PRICE_CHECKERS
-    ].arguments.map((arg) =>
-      arg.convertInput(data[arg.name], data.tokenBuy.decimals),
-    );
-
-    await sendTransactions([
-      {
-        type: TRANSACTION_TYPES.ERC20_APPROVE,
-        tokenAddress: data.tokenSell.address,
-        spender: MILKMAN_ADDRESS,
-        amount: sellAmountBigInt,
-      },
-      {
-        type: TRANSACTION_TYPES.MILKMAN_ORDER,
-        tokenAddressToSell: data.tokenSell.address,
-        tokenAddressToBuy: data.tokenBuy.address,
-        toAddress: data.receiverAddress,
-        amount: sellAmountBigInt,
-        priceChecker: data.priceChecker,
-        args: priceCheckersArgs,
-      },
-    ]);
-    router.push(`/milkman/${network}`);
-  }
+  const [orderOverviewData, setOrderOverviewData] = useState<FieldValues>();
+  const [priceCheckerData, setPriceCheckerData] = useState<FieldValues>();
```

Comment:
Think I wouldn't create a state form this... because one of the points of using react hook form is to deal with the state within the form

Reply:
we are using this to combine the form data, correct?

Reply:
a blocker here is the priceChecker form changing for each price checker